### PR TITLE
glpicrypt-key creation and persistence

### PR DESCRIPTION
**Pull request by Anselm98**
* Automatically create glpicrypt.key file
* Persists the glpicrypt.key file in "data" volume

### CODE REVIEW

File: charts/itsm-ng/templates/deployment.yaml [hunk 1]
```diff
@@ -26,6 +26,18 @@ spec:
...
+
+
+      initContainers:
+        - name: glpicryptkey-init
+          image: alpine:3.19.1
+          command: ['sh', '-c', 'mkdir -p /var/www/html/config ; touch /var/www/html/config/glpicrypt.key']
+          volumeMounts:
+            - name: data
+              mountPath: /var/www/html/config/
+              subPath: config
```
Comment (bmalynovytch) [inline charts/itsm-ng/templates/deployment.yaml]:
```suggestion
          command: ['sh', '-c', 'mkdir -p /data/config ; touch /data/config/glpicrypt.key']
          volumeMounts:
            - name: data
              mountPath: /data
```

**Review by bmalynovytch — DISMISSED** [review]

**Comment by bmalynovytch** [comment]
@Anselm98 you need to update the chart version

**Comment by bmalynovytch** [comment]
@Anselm98 you need to also update the image.tag defalut value if you update the appVersion

**Comment by Anselm98** [comment]
image tag changed to 1.6.0-001 in values.yaml

File: charts/itsm-ng/Chart.yaml [hunk 2]
```diff
@@ -15,13 +15,13 @@ type: application
...
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.6
+version: 0.1.7
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.3.0"
+appVersion: "1.6.0"
```
Comment (bmalynovytch) [inline charts/itsm-ng/Chart.yaml]:
```suggestion
appVersion: "1.3.0"
```

File: charts/itsm-ng/values.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@ image:
   repository: admantic/itsm-ng
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "1.3.0-005"
+  tag: "1.6.0-001"
```
Comment (bmalynovytch) [inline charts/itsm-ng/values.yaml]:
```suggestion
  tag: "1.3.0-005"
```

**Review by bmalynovytch — APPROVED** [review]

**Pull request merged**